Lesson007/Task 5 HW07.BookStore/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace Task5HW07.BookStore
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int[,,] arreyBooks =
+            {
+                {
+                   {09, 12, 05, 15},
+                   {10, 08, 07, 11},
+                   {35, 12, 12, 16},
+                   {15, 24, 67, 89}
+                },
+                {
+                   {88, 76, 13, 33},
+                   {11, 53, 61, 14 },
+                   {56, 44, 22, 17 },
+                   {11, 98, 13, 14 }
+                },
+                {
+                   {29, 27, 77, 14},
+                   {78, 03, 55, 17},
+                   {48, 19, 68, 80},
+                   {68, 41, 14, 07}
+                }
+            };
+
+            int max = arreyBooks[0, 0, 0], min = arreyBooks[0, 0, 0];
+
```

Comment:
usually need to use int.MaxValue as max value and int.MinValue as min value

Reply:
Fixed